presto-main/src/main/java/com/facebook/presto/server/ServerConfig.java

```diff
@@ -201,4 +202,16 @@ public ServerConfig setClusterStatsExpirationDuration(Duration clusterStatsExpir
         this.clusterStatsExpirationDuration = clusterStatsExpirationDuration;
         return this;
     }
+
+    public boolean isNestedDataSerializeEnabled()
```

Comment:
Here and elsewhere: `serialize` => `serialization`

Reply:
Done!